addon/components/power-select-infinity/create-message.hbs

```diff
@@ -0,0 +1,4 @@
+{{#if (and @select.searchText (not @extra.isSearching))}}
+    {{! template-lint-disable no-bare-strings }}
+    <div class="ember-power-select-option border-bottom">Press enter to create the group "{{@select.searchText}}"</div>
```

Comment:
If I'm understanding this correctly I think the text inside this element should be replaced like:

```hbs
<div class="ember-power-select-option border-bottom">{{this.createMessage}} "{{@select.searchText}}"</div>
```

```ts
get this.createMessage() {
    return this.args.createMessage ?? "Press enter to create the group ";
}

Or even possibly a function arg to the component where the function takes the searchText as an argument and returns a string that would be rendered here